HubSpot/hubspot-cli
# chore: Update all extensions to .ts

## Description and Context



- Update all files to `.ts` extensions and add a `// @ts-nocheck` comment to the top of the file
- Add `ts-jest` as a dev dep to be used as a `jest` preset

## Testing steps
- Run `yarn build`
- Run the ATs 
- Run the unit tests 
- Spot check a few commands
